autoreact/ktp_xarray/xarray_wrappers.py

```diff
@@ -18,75 +17,106 @@ def from_data(temps, press, rates):
 
 # Getters
 def get_pressures(ktp):
-    """
-    Gets the pressure values
-    """
+    """Gets the pressure values"""
 
     return ktp.pres.data
 
 
 def get_temperatures(ktp):
-    """
-    Gets the temperature values
-    """
+    """Gets the temperature values"""
 
     return ktp.temp.data
 
 
 def get_values(ktp):
```

Comment:
This function and the next three could be combined into a single, generic function for accessing values:
```
def values(ktp, pres=None, temp=None):
    if pres is None and temp is None:
        return ktp.values
    elif temp is None:
        return ktp.sel(pres=pres)
    elif pres is None:
        return ktp.sel(temp=temp)
    return ktp.sel(temp=temp, pres=pres)
```
I have found this to be generally more convenient (and easier from a user perspective) than defining separate functions for closely related things